akka-actor-typed-tests/src/test/scala/akka/actor/typed/InterceptSpec.scala

```diff
@@ -55,6 +55,7 @@ object InterceptSpec {
         val wrapped = msg match {
           case c: Command          => InternalProtocol.WrappedCommand(c)
           case r: ExternalResponse => InternalProtocol.WrappedExternalResponse(r)
+          case unexpected          => throw new RuntimeException(s"Unexpected: $unexpected")
```

Comment:
this looks much better than

```scala
case _ => throw new RuntimeException() // compiler exhaustiveness check pleaser
```

We never know when things get broken. It will be good if can retain some context when unexpected things happen.

Reply:
Yeah, I tried to be deliberate in where it is just unlikely to happen (and include info about what was matched like that) and where I can know it will never happen but the compiler just can't know that.